### ISSUE TITLE
Editing header, footer and other Elementor made parts anywhere

### ISSUE THREAD

**Issue by bentalgad** [elementor/elementor#4985]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**Editing the header and footer can be so much easier**
A lot of times you want to edit some little thing in the header or the footer when you working on one of your site pages (especially when working on the homepage maybe).

**It would be so great if you can edit it on the place on any page it shows up on**
So basically just let us edit all parts made by Elementor that show up on the page were editing without leaving that page.  

**And if not... at least**
Give is some quick link for editing those parts if we click on them while editing some page. but of course that's a less nicer solution.

Thanks for that great plugin! Hope it would be come better and better soon.

**Comment by bainternet** [comment]
Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.

but we do have plans to allow easier access.

Thanks for the feedback.

**Comment by bentalgad** [comment]
I know that's not supported. that's the idea of "Feature request" isn't it?

**Comment by ArekZw** [comment]
+1

**Comment by Wereldsmuller** [comment]
> Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.
> 
> but we do have plans to allow easier access.
> 
> Thanks for the feedback.

Sounds interesting. Anything you could already share about easier access and what it would look like?

**Comment by bentalgad** [comment]
"Brizy" builder has this ability. "Jetelements" has one step forward this idea where they give you a "edit" button on a template used inside other widget like "Jettabs" and if you press that button you get iframe with another Jlementor editor with that template to edit, but of course that's not the direction i think it should go for, the brizy approach is best i think, where you just can edit any template from any page it appears on. so for example if you edit the page you as your homepage and then you want to edit something in the header which sometimes can be because you want to make some design elements match betaween the homepage main content and the header for example and then you have to go to the header template, edit it, save it, go back to your homepage, and only then you see the header change in the context of that page, but if would be able to just make a change on the header when in your homepage it would make your work so fluent and much more creatively free.

**Comment by m2sahin** [comment]
+1

**Comment by bentalgad** [comment]
Still Waiting.

**Comment by jazir5** [comment]
This would be good to have, +1

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.9.1

Feel free to update

Thanks!

**Comment by bentalgad** [comment]
Thanks! that's wonderful!

**Comment by francescodicandia** [comment]
Awesome!!!
